docs/pressable.md

```diff
@@ -136,41 +138,41 @@ Duration (in milliseconds) from `onPress` before `onLongPress` is called.
 
 Whether the press behavior is disabled.
 
-| Type    | Required |
-| ------- | -------- |
-| boolean | No       |
+| Type    | Required | Default |
+| ------- | -------- | ------- |
+| boolean | No       | `false` |
 
 ### `hitSlop`
 
 Sets additional distance outside of element in which a press can be detected.
 
-| Type       | Required |
-| ---------- | -------- |
-| RectOrSize | No       |
+| Type       | Representation                                                                         | Required |
+| ---------- | -------------------------------------------------------------------------------------- | -------- |
+| RectOrSize | number <hr/> object `{ bottom: ?number, left: ?number, right: ?number, top: ?number }` | No       |

```

Comment:
Each key is optional. As worded right now it makes it seem like every key is required, but takes `null | undefined | number`. 

I think it should be:
```
number <hr/> object `{ bottom?: number, left?: number, right?: number, top?: number }`
```

Or if we are actually using Flow syntax like we have in the codebase, technically both sides need question marks because it *does* let a key be defined and set to `null | undefined` as well.
 
```
number <hr/> object `{ bottom?: ?number, left?: ?number, right?: ?number, top?: ?number }`
```

Reply:
That's a good remark, thank you! I like the second version because it's true to the code. But from the discussions and polls it seems that quite many developers are not familiar with the Flow syntax. Unfortunately (due to MD files parsing rules) it's not possible to put code blocks inside tables which restrain the possibilities to format code there.

This is the small preview which shows what can be achieved using some more CSS, but I'm not sure if it isn't too much information (there are two representations for key type for the comparison):
<img width="627" alt="Annotation 2020-05-15 144730" src="https://user-images.githubusercontent.com/719641/82052165-7b4e4800-96bb-11ea-84fc-2dfdb000db74.png">

Is it a better solution? What's your opinion?

Reply:
Not a fan of either of these, personally. Think we should go with an Object Type page, [ala MDN](https://developer.cdn.mozilla.net/en-US/docs/Web/API/Web_Animations_API/Keyframe_Formats). Better to be explicit, and we're going to have to do this with other objects on the page as well, so...

Reply:
<img width="1355" alt="Screenshot 2020-05-15 at 15 08 14" src="https://user-images.githubusercontent.com/236306/82060196-f49b6a00-96be-11ea-9385-c40d9dbfde2f.png">
This is how that could look. Making a separate PR!